Symfony/CS/Tests/Fixer/PSR2/BracesFixerTest.php

```diff
@@ -32,6 +32,20 @@ public function provideFixControlContinuationBracesCases()
         return array(
             array(
                 '<?php
+class Foo
+{
+    public function A()
+    {
+        ?>
+        Test<?php echo $foobar; ?>Test
+        <?php
+        $a = 1;
```

Comment:
I understand, I propose to doc. this as a known edge case and not fix it, i.e. choose speed over fixing an edge case.